Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM python:3.8-slim-buster as base
+
+# Set working directory
+WORKDIR /app
+
+# Set environment values
+ENV PATH="${PATH}:/root/.poetry/bin"
+
+#Install Curl
+RUN apt-get update && apt-get install -y curl
+
+# Install Poetry
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+# Install poetry files
+COPY poetry.lock /app
+COPY pyproject.toml /app
+COPY poetry.toml /app
+RUN poetry install
+
+#Copy app code
+COPY todo_files /app
+
+# run entrypoint
+COPY entrypoint.sh /app
+RUN chmod 755 /app/entrypoint.sh
+CMD /app/entrypoint.sh
```

Comment:
This implies that this isn't a multistage build

Reply:
Have removed these and uncommented the mulitstage parts below